deploy/ray-operator/controllers/common/pod.go

```diff
@@ -85,15 +85,6 @@ func buildContainer(conf *PodConfig) corev1.Container {
 		Resources:       conf.Extension.Resources,
 		VolumeMounts:    conf.Extension.VolumeMounts,
 		ImagePullPolicy: conf.RayCluster.Spec.ImagePullPolicy,
-		Ports: []corev1.ContainerPort{
-			{
-				ContainerPort: int32(defaultRedisPort),
-				Name:          "redis",
-			},
-			{
-				ContainerPort: int32(defaultHTTPServerPort),
-				Name:          "http-server",
-			},
-		},
+		Ports:           conf.Extension.Ports,
```

Comment:
Will `Ports` have a default configuration if user didn't provide them?

Reply:
Added default ports when not provided.